build.sbt

```diff
@@ -3567,7 +3571,7 @@ ThisBuild / buildEngineDistributionNoIndex := {
 lazy val runEngineDistribution =
   inputKey[Unit]("Run or --debug the engine distribution with arguments")
 runEngineDistribution := {
-  buildEngineDistribution.value
+  buildEngineDistributionNoIndex.value
```

Comment:
Once this is integrated, the `runEngineDistribution` command is going to be way faster. Thanks @Akirathan for the idea.